src/pushNotifications.js

```diff
@@ -0,0 +1,9 @@
+import PushNotification from 'react-native-push-notification'
+import { FetchResult } from '@react-native-community/push-notification-ios'
+
+PushNotification.configure({
```

Comment:
this for debugging? is these when user clicks on notification?
we will probably need to open the relevant feed item here no?

Reply:
What do you mean with the relevant feed item? This is called when a remote or local notification is opened or received. The functionality we have is open the app when the user touches the notification.

Reply:
> What do you mean with the relevant feed item? 
Probably when notification is clicked, we should open user's feed and scroll to notification we've clicked
@sirpy, am i right ?

Reply:
a notification is about a feed item, so when user opens the app we expect it to open the relevant item